### DISCUSSION TITLE
Multi-Step Forecasting Possible?

### DISCUSSION THREAD

**Discussion by ghost** [mljar/mljar-supervised#321] [Q&A] [answered]
Is it possible to perform multi-step forecasting using mljar; if possible are there examples? Example is given 100 current data points, predict future 10 data points.

**Comment by pplonski** [comment, marked as answer]
1. What do you mean by multi-step?
2. Do you want to analyze time-series data? For example stock data to predict the future prices?

**Reply by ghost** [reply]
Yes, is that a feature?

**Reply by pplonski** [reply]
Yes, it is possible. But the data should be manually prepared. Input data should be stationary.

**Reply by ghost** [reply]
Is there a function for this? Or do I need to manually recreate training data using newly predicted data?

**Reply by pplonski** [reply]
You need to do this manually.
